services/chrome-web-store/chrome-web-store-base.js

```diff
@@ -1,22 +1,19 @@
 'use strict'
 
-const chromeWebStore = require('chrome-web-store-item-property')
+const ChromeWebStore = require('webextension-store-meta/lib/chrome-web-store')
```

Comment:
For consistency with the other Chrome classes, it's probably best to keep it as lower camelcase, i.e. `chromeWebStore `.

Reply:
> For consistency with the other Chrome classes

What does it mean? The uppercase is because `ChromeWebStore` is a class. `ChromeWebStore.load` is a static method.

Reply:
I think I got a little confused. Let's ignore this one!